src/manager.cpp

```diff
@@ -179,4 +186,57 @@ sdbusplus::async::task<>
     co_return;
 }
 
+// NOLINTNEXTLINE
+sdbusplus::async::task<void> Manager::startFullSync()
+{
```

Comment:
I think tracing the full sync start would be interesting data to have.  Then we could have to count backwards in the journal based  on how long it took to find it.